src/Lib/Managers/SelectedBoundManager.vala

```diff
@@ -43,6 +43,13 @@ public class Akira.Lib.Managers.SelectedBoundManager : Object {
     private double initial_width;
     private double initial_height;
 
+    // drag move state
+    private double initial_drag_press_x;
+    private double initial_drag_press_y;
+    private bool   initial_drag_item_registered = false;
```

Comment:
we don't align attribute names, always use 1 space after the type declaration.
Also, what about calling this `initial_drag_registered`?